src/lua.cpp

```diff
@@ -242,6 +245,80 @@ static int disassembleTeakThunk(lua_State* L) {
 	return 1;
 }
 
+#ifndef __ANDROID__
+// Haptics functions
+
+namespace Haptics {
+	std::unique_ptr<ExternalHapticsManager> hapticsManager = nullptr;
+
+	static int initHapticsThunk(lua_State* L) {
```

Comment:
Static functions in global scope are deprecated in favor of putting the function in an unnamed namespace instead